cognee/infrastructure/databases/vector/create_db_and_tables.py

```diff
@@ -0,0 +1,15 @@
+from ..relational.ModelBase import Base
+from .get_vector_engine import get_vector_engine, get_vectordb_config
+from sqlalchemy import text
+
+async def create_db_and_tables():
```

Comment:
We could try to move this inside pgvector adapter.

Reply:
Done